data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -293,7 +296,7 @@ then
 			--mdns-publisher-image="${MDNS_PUBLISHER_IMAGE}" \
 			--haproxy-image="${HAPROXY_IMAGE}" \
 			--baremetal-runtimecfg-image="${BAREMETAL_RUNTIMECFG_IMAGE}" \
-			--cloud-config-file=/assets/manifests/cloud-provider-config.yaml \
+			--cloud-config-file=/assets/config-bootstrap/cloud-provider-config-generated.yaml \
```

Comment:
based on the change suggested in https://github.com/openshift/installer/pull/3831/files#r463786564
```suggestion
			--cloud-config-file=/assets/manifests/cloud-provider-config-generated.yaml \
```